src/classes/Language.as

```diff
@@ -196,6 +196,10 @@ package classes
                 }
                 indexed[data[lang]["_index"]] = lang;
             }
+
+            data["us"]["options_receptor_animations"] = "Show Receptors Animations";
+            data["us"]["options_receptor_speed"] = "Receptor Flash Speed:";
+            
```

Comment:
These should probably be strings from the language database

Reply:
Will do